mysql.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM mysql:latest
+
+ENV MYSQL_ROOT_PASSWORD=1234
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
+
+EXPOSE 3306
+
+VOLUME /var/lib/mysql
+
+COPY init.sql /docker-entrypoint-initdb.d/
```

Comment:
![image](https://github.com/mate-academy/devops_todolist_docker_core_task_2_volumes/assets/87720583/6e4f9153-ca03-4905-acfa-ab992de11248)
Add Eof(End-of-file), need to add empty line at the end